demo_serial/src/main/java/com/novoda/downloadmanager/demo/serial/MainActivity.java

```diff
@@ -40,6 +41,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     private void setupDownloadingExample() {
         Uri uri = Uri.parse(BIG_FILE);
+        final DownloadBatch batch = new DownloadBatch("Title woo", "Description lols", BBC_COMEDY_IMAGE);
```

Comment:
These were just for testing, but yes, you're right. 
Do you think we should make a separate demo app when all this feature is implemented?